controllers/controller.js

```diff
@@ -1,13 +1,13 @@
-const { selectCategories, selectReviewById } = require("../models/model");
+const {
+  selectCategories,
+  selectReviewById,
+  selectUsers,
+} = require("../models/model");
 
 exports.getController = (req, res) => {
-  selectCategories()
-    .then((categories) => {
-      res.status(200).send({ categories });
-    })
-    .catch((err) => {
-      console.log(err);
-    });
+  selectCategories().then((categories) => {
+    res.status(200).send({ categories });
+  });
```

Comment:
missing a catch block here, it seems to have been removed, probably due to some merging changes but make sure it gets added back